src/libertem/web/config.py

```diff
@@ -23,3 +23,37 @@ async def get(self):
         msg = Message(self.state).config(config=self.state.get_config())
         log_message(msg)
         self.write(msg)
+
+
+class ClusterDetailHandler(tornado.web.RequestHandler):
+    def initialize(self, state: SharedState, event_registry):
+        self.state = state
+        self.event_registry = event_registry
+
+    async def get(self):
+        executor = self.state.executor_state.get_executor()
+        workers = await executor.get_available_workers()
+        details = {}
+
+        for worker in workers:
+            if worker.name.startswith("tcp"):
+                host_name = worker.host
+                resource = 'cpu'
+            else:
+                host_name = '-'.join(worker.name.split('-')[:-2])
+                resource = worker.name.split('-')[-2]
```

Comment:
What do you think about putting the host name / resource split into the worker class?

Reply:
That would be more meaningful. Is there any particular reason for using the current format ?

Reply:
I haven't changed this yet.